src/tjbot.js

```diff
@@ -636,74 +636,65 @@ class TJBot {
-     *                     "score": 0.771241,
-     *                     "tone_id": "sadness",
-     *                     "tone_name": "Sadness"
-     *                 },
-     *                 {
-     *                     "score": 0.687768,
-     *                     "tone_id": "analytical",
-     *                     "tone_name": "Analytical"
-     *                 }
-     *             ]
-     *         },
-     *         {
-     *             "sentence_id": 2,
-     *             "text": "We have a competitive product, but we need to do a better job of selling it!",
-     *             "tones": [{
-     *                 "score": 0.506763,
-     *                 "tone_id": "analytical",
-     *                 "tone_name": "Analytical"
-     *             }]
-     *         }
-     *     ]
+     *      "usage": {
+     *          text_units": 1,
+     *          "text_characters": 37,
+     *          "features": 1
+     *      },
+     *      "language": "en",
+     *      "emotion": {
+     *          "targets": [
+     *              {
+     *                  "text": "apples",
+     *                      "emotion": {
+     *                      "sadness": 0.028574,
+     *                      "joy": 0.859042,
+     *                      "fear": 0.02752,
+     *                      "disgust": 0.017519,
+     *                      "anger": 0.012855
+     *                      }
+     *              }
+     *          ],
+     *      "document": {
+     *          "emotion": {
+     *              "sadness": 0.32665,
+     *              "joy": 0.563273,
+     *              "fear": 0.033387,
+     *              "disgust": 0.022637,
+     *              "anger": 0.041796
+     *          }
+     *      }
+     *  }
+
      * }
-     * @see {@link https://cloud.ibm.com/apidocs/tone-analyzer?code=node#tone|Tone Analyzer} documentation provides details on the response object.
+     * @see {@link https://cloud.ibm.com/apidocs/natural-language-understanding?code=node#emotion|Natural Language Understanding} documentation provides details on the response object.
      * @async
      */
     async analyzeTone(text) {
         this._assertCapability(TJBot.CAPABILITIES.ANALYZE_TONE);
 
         const params = {
-            toneInput: { text },
-            contentType: 'application/json',
+            text: text,
+            language: 'en',
+            features: {
+                emotion: {
+                  targets: [
+                    text
+                  ]
+                }
+              }
         };
 
         try {
-            const body = await this._toneAnalyzer.tone(params);
+            const body = await this._toneAnalyzer.analyze(params);
             winston.silly(`response from _toneAnalyzer.tone(): ${body}`);
             return body.result;
         } catch (err) {
-            winston.error(`the ${TJBot.SERVICES.TONE_ANALYZER} service returned an error.`, err);
+            console.log("err: ", err);
```

Comment:
when this actually emits something, it may be difficult to reverse-engineer that it was emitted from this line. also, we should probably not be doing `console.log` given that we're using `winston` for debug logging, right? :)